src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -30,12 +30,12 @@ using namespace mlpack::ann;
  */
 template<typename MatType = arma::mat, typename ModelType>
 void TestNetwork(ModelType& model,
-                 MatType& trainData,
-                 MatType& trainLabels,
-                 MatType& testData,
-                 MatType& testLabels,
-                 const size_t maxEpochs,
-                 const double classificationErrorThreshold)
+                  MatType& trainData,
```

Comment:
```suggestion
                 MatType& trainData,
```

Hey @heisenbuug , indentation issue still persists here and at other places.